test_assets/PrimvarInterpolation/README.md

```diff
@@ -0,0 +1,221 @@
+# Primvar Interpolation
+
+![annotated primvar interpolation screenshot](images/annotated_results_storm.png)
```

Comment:
The images are not showing up for me in the "display rich text" mode, not sure if they will work once the PR is merged.

Reply:
You can see how it'll look like when you click the "..." button on the top right and then "View File". It'll go to [this link](https://github.com/usd-wg/assets/blob/0d572735561950c44870d2706f3489f53dc0bc76/test_assets/PrimvarInterpolation/README.md).